wallit/dbio.go

```diff
@@ -64,35 +64,44 @@ func (w *Wallit) AddPorTxoAdr(kg portxo.KeyGen) error {
 	})
 }
 
-// AdrDump returns all the addresses in the wallit.
-// currently returns 20 byte arrays, which
-// can then be converted somewhere else into bech32 addresses (or old base58)
-func (w *Wallit) AdrDump() ([][20]byte, error) {
-	var i, last uint32 // number of addresses made so far
-	var adrSlice [][20]byte
+// getNumberOfKeys retrieves the number of p2pkh keys in the database and check whether
+// the maximum number of keys was reach or not.
+func (w *Wallit) GetNumberOfKeys() (uint32, error) {
+	// Could be placed somewhere else, perhaps it's conf ?
+	var maxKeys = uint32(1 << 20) // the reasonable limit of addresses belonging to one Wallit
```

Comment:
Well, I guess *technically* there's no upper limit on the number of addresses, but this would help :)

Reply:
I haven't found any reasons for why such a limit was enforced. The main point here is to stay coherent between functions checking this limit, and not have magic numbers all around the place.

Reply:
Agreed